src/Employer/UnitTests/EmployerProvideFeedback/Controllers/HomeControllerTests.cs

```diff
@@ -114,5 +96,65 @@ public async void SessionSurvey_Exists_ShouldNotCreateNewSurvey()
             // Assert
             _sessionServiceMock.Verify(mock => mock.Set(uniqueCode.ToString(), It.IsAny<SurveyModel>()), Times.Never);
         }
+
+        private DefaultHttpContext GetHttpContext()
+        {
+            var _context = new DefaultHttpContext()
+            {
+                User = new ClaimsPrincipal(new ClaimsIdentity(new Claim[]
+                {
+                    new Claim(ClaimTypes.NameIdentifier, GetSurveyModel().UserRef.ToString()),
+                }))
+            };
+            return _context;
```

Comment:
For brevity, this can become one statement without the variable declaration:
```suggestion
            return new DefaultHttpContext()
            {
                User = new ClaimsPrincipal(new ClaimsIdentity(new Claim[]
                {
                    new Claim(ClaimTypes.NameIdentifier, GetSurveyModel().UserRef.ToString()),
                }))
            };
```

Reply:
done